README.md

```
@@ -1,6 +1,22 @@
 # **WPF DirectX Extensions**
 WPF DirectX Extensions allow you to easily host DirectX 10 and DirectX 11 content in WPF applications.
 
+Supported .NET versions
```

markdownlint-cli2 (0.23.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)